src/utils/settings.cpp

```diff
@@ -37,11 +37,13 @@ namespace
 const auto file_extension = QStringLiteral("conf");
 const auto daemon_root = QStringLiteral("local");
 const auto petenv_name = QStringLiteral("primary");
+const auto autostart_default = QStringLiteral("false");
```

Comment:
Hmm I wonder if this should be `true` instead? This way when the user installs and runs the GUI, it will autostart from then on, until they disable it. We should be able to add a toggle of some kind to the tray menu, too.

Reply:
It won't unfortunately, not until either CLI or GUI are run manually. Here is the msg from that commit:

```
[settings] Disable autostart by default

Otherwise, the flag would not be respected until the user ran a client.
This way, autostarting requires a `multipass set`, at which point the
autostart infrastructure is setup. This also accommodates the plan to
implement the autostart setting with the underlying platform 
rather than a plain flag.
```

Reply:
Not convinced, on Windows we have the installer where we can enable autostart (long-term - conditionally). That's likely the same on macOS, really only snaps are problematic here, and I actually think they will fix it long term.

When we implement it with the platform, the default here won't matter anyway.

Reply:
Are you sure? If set to true now, how do we handle it in the platform implementation? How will we be able to have a default of `true` then?

Reply:
On Windows, you add the registry key unconditionally already. The config key becomes moot and `multipass.gui --autostart` won't look at it, if called it will just start. Migration from the key to platform implementation will be a bit awkward, but I think solvable.

Reply:
Sorry, I seem to have edited out "When we move to platform implementation of this…".

I agree it may lead to inconsitent behaviour today (it will auto-start upon installation on Windows and hopefully macOS, but not on Linux), but as soon as you run anything `multipass…` it will sort itself out.